app/components/users/UserAdd.js

```diff
@@ -0,0 +1,72 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import toastr from 'toastr';
+import * as userActions from '../../actions/userActions';
+import UserForm from './UserForm';
+
+class UserAdd extends React.Component {
+    constructor(props, context) {
+        super(props, context);
+
+        this.state = {
+            user: Object.assign({}, props.user),
+            saving: false
+        };
+
+        this.handleSave = this.handleSave.bind(this);
+    }
+
+    handleSave(user) {
+        this.setState({ saving: true });
+        this.props.actions.createUser(user)
+            .then(() => {
+                toastr.success('User created successfully');
+                this.setState({ saving: false });
```

Comment:
Thanks for the feedback folks. Actually I already did that improvement in another branch @juampick hehe. Maybe I will do all the improvements and merge it with this one. Not sure yet.

Answering @AlphaGit's question: a quick way is to look for a `connect` keyword. Those are the container-kind of components. I'm getting used to myself too so I believe you should consider those as page-level component (at least for the majority of scenarios).